specification/_types/aggregations/AggregationContainer.ts

```diff
@@ -110,7 +111,7 @@ export class AggregationContainer {
   /**
    * @variant container_property
    */
-  meta?: Dictionary<string, UserDefinedValue>
+  meta?: Metadata
```

Comment:
@swallez can you confirm that this change has no impact on the variant annotation?

Reply:
Thank you double checking!